ports/xnnpack/portfile.cmake

```
@@ -5,20 +5,24 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/XNNPACK
-    REF 51a987591a6fc9f0fc0707077f53d763ac132cbf # 2022-12-22
-    SHA512 1742ad4de3aa1a67cd5a3cbf15d82afc2cc8569a2abac92542f748a440ef37fef4e9d9c302b7b2813e3176934b637c761c1c0956d20ab1ba28b280d9b121d91b
+    REF 87ee0b46b834f67bad9025d4a82ed5654f3403d3 # 2024-18-20
```

The date string is invalid. There is no 18th month.

---

```suggestion
    REF 87ee0b46b834f67bad9025d4a82ed5654f3403d3 # 2024-08-20
```

Based on https://github.com/google/XNNPACK/commit/87ee0b46b834f67bad9025d4a82ed5654f3403d3